tools_for_todoist/models/google_calendar.py

```diff
@@ -88,7 +88,7 @@ def _process_sync(self):
                 created_events.append(new_event)
             else:
                 self._events[event['id']].update_from_raw(event)
-                updated_events.append(event)
+                updated_events.append(self._events[event['id']])
```

Comment:
we should probably create a dummy copy of the old event, and save a pair, so future consumers can check exactly what changed. Events in google get updated for various reasons (including someone responded yes to them).